### PR TITLE
feat: pmt indication on position card

### PR DESCRIPTION
**Pull request by jardt**
Org position cards can now show new property on positions (isProjectManagementTeam).
It will appear as text below exiting icons on left hand side. Warning version will be displayed if property bag does not contain true value for field allowExternalProjectManagementTeam. Which is a setting allowing the allocated person to be part of PMT regardless of external status.

### CODE REVIEW

**Review by MarenSpangberg — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@equinor/fusion-components",
-    "version": "2.12.2",
+    "version": "2.13.0",
```
Comment (eikeland) [inline package.json:3]: Please us a patch version bump for this release

File: src/components/org/PositionCard/components/PositionIconPhoto.tsx [hunk 2]
```diff
@@ -25,6 +26,7 @@ type PositionPhotoIconProps = {
     anonymize?: boolean;
     inline?: boolean;
     highlightTaskOwner?: boolean;
+		showPmt?: boolean;
```
Comment (eikeland) [inline src/components/org/PositionCard/components/PositionIconPhoto.tsx:29]: indentation

File: src/components/org/PositionCard/components/PositionIconPhoto.tsx [hunk 3]
```diff
@@ -37,13 +39,22 @@ const PositionPhotoIcon: FC<PositionPhotoIconProps> = ({
     anonymize,
     inline,
     highlightTaskOwner,
+		showPmt,
```
Comment (eikeland) [inline src/components/org/PositionCard/components/PositionIconPhoto.tsx:42]: indentation

File: src/components/org/PositionCard/components/PositionIconPhoto.tsx [hunk 4]
```diff
@@ -37,13 +39,22 @@ const PositionPhotoIcon: FC<PositionPhotoIconProps> = ({
...
 }) => {
+
+    const isRotating = rotationInstances.length > 0;
+    const isTaskOwner = position.isTaskOwner;
+    const isPmt = position.isProjectManagementTeam && showPmt;
+
+		//show pmt as warning if allowExternalProjectManagementTeam is not enabled but non employee is assigned regardless
+		const pmtExternalWarning = isPmt && !(position.properties as Record<string, unknown>).allowExternalProjectManagementTeam && currentInstance.assignedPerson && currentInstance.assignedPerson?.accountType !== 'Employee'
```
Comment (eikeland) [inline src/components/org/PositionCard/components/PositionIconPhoto.tsx:50]: Indentation. 
if type `position.properties` are used other places consider extending the type with `allowExternalProjectManagementTeam`.

File: src/components/org/PositionCard/components/PositionIconPhoto.tsx [hunk 5]
```diff
@@ -37,13 +39,22 @@ const PositionPhotoIcon: FC<PositionPhotoIconProps> = ({
...
+
     const containerRef = useRef<HTMLDivElement>(null);
     const linkedRef = useTooltipRef('Linked', 'below');
     const rotatingRef = useTooltipRef('Rotating', 'below');
 
-    const isRotating = rotationInstances.length > 0;
-    const isTaskOwner = position.isTaskOwner;
+		const pmtTooltip = pmtExternalWarning ? <p> PMT role is not enabled for external personnel on this position.<br /> You can turn on this setting while editing the position. </p>: <p>Personell allocated to this position has PMT (Project Management Team) role,<br/> giving them access to restricted information.</p>;
```
Comment (eikeland) [inline src/components/org/PositionCard/components/PositionIconPhoto.tsx:56]: indentation

File: src/components/org/PositionCard/components/PositionIconPhoto.tsx [hunk 6]
```diff
@@ -99,6 +110,11 @@ const PositionPhotoIcon: FC<PositionPhotoIconProps> = ({
                             <SyncIcon color={styling.colors.blackAlt2} height={16} width={16} />
                         </span>
                     )}
+                    {isPmt && (
+                        <span ref={pmtRef}>
+														{pmtExternalWarning ? (<div className={styles.pmtWarning}><s className={styles.pmtText}>PMT</s><ErrorIcon outline /> </div>) :  (<p className={styles.pmtText}>PMT</p>)}
```
Comment (eikeland) [inline src/components/org/PositionCard/components/PositionIconPhoto.tsx:115]: indentation

File: src/components/org/PositionCard/index.tsx [hunk 7]
```diff
@@ -46,6 +46,7 @@ type PositionCardProps = {
     anonymize?: boolean;
     inline?: boolean;
     highlightTaskOwner?: boolean;
+		showPmt?: boolean;
```
Comment (eikeland) [inline src/components/org/PositionCard/index.tsx:49]: Indentation

File: src/components/org/PositionCard/index.tsx [hunk 8]
```diff
@@ -105,6 +106,7 @@ const PositionCard: React.FC<PositionCardProps> = ({
     anonymize,
     inline,
     highlightTaskOwner,
+		showPmt,
```
Comment (eikeland) [inline src/components/org/PositionCard/index.tsx:109]: Indentation

File: src/components/org/PositionCard/index.tsx [hunk 9]
```diff
@@ -187,6 +189,7 @@ const PositionCard: React.FC<PositionCardProps> = ({
                     anonymize={anonymize}
                     inline={inline}
                     highlightTaskOwner={highlightTaskOwner}
+										showPmt={showPmt}
```
Comment (eikeland) [inline src/components/org/PositionCard/index.tsx:192]: Indentation

File: src/components/org/PositionCard/styles.less [hunk 10]
```diff
@@ -112,14 +112,34 @@
                     justify-content: center;
                 }
             }
+						.pmtText {
```
Comment (eikeland) [inline src/components/org/PositionCard/styles.less:115]: Indentation

**Review by eikeland — CHANGES_REQUESTED** [review]
Improve formatting, otherwise great 👍🏻

File: package.json (hunk 1 above)
Comment (jardt) [reply package.json]: fixed in [19e4d53](https://github.com/equinor/fusion-components/pull/1691/commits/19e4d53af5ba0c006bf55cb90c888bf6d27fdb69)

File: src/components/org/PositionCard/components/PositionIconPhoto.tsx (hunk 4 above)
Comment (jardt) [reply src/components/org/PositionCard/components/PositionIconPhoto.tsx]: Will consider, only used in one place in org admin. Needs to be done in fusion-api package

**Comment by jardt** [comment]
> Improve formatting, otherwise great 👍🏻

Sorry editor did not autoformat but meant to run format script before asking for PR,
now formatted

**Pull request merged**